kuka_lbr_iiwa_support/launch/test_lbr_iiwa_14_r820.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <include file="$(find kuka_lbr_iiwa_support)/launch/load_lbr_iiwa_14_r820.launch" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
+    <param name="use_gui" value="true" /> 
+  </node>
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find industrial_robot_client)/config/robot_state_visualize.rviz" required="true" />
+</launch>

```

Comment:
On my local clone, this file appears to have Windows line-endings. Could you fix that?